### PR TITLE
Add Github actions

### LINKED ISSUE #6: GitHub Actions 自动构建
额外可能需要注意的地方：
- 密码。如果构建产物最终公开，需要参考 https://wiki.archlinux.org/title/SHA_password_hashes 检查并提升密码强度。

### PR DESCRIPTION
**Pull request by RetiredWorld**
Add Github actions to auto build liimstrap image, close #6.

### CODE REVIEW

File: .github/workflows/build-liimstrap.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - name: Pack to squashfs file
+      run: docker run --privileged --cap-add=SYS_ADMIN --rm -v /srv/dest:/srv/dest -e ROOT_PASSWORD=${{ secrets.LIIMSTRAP_ROOT_PASSWORD }} -e SQUASHFS=true ustclug/liimstrap:liims-2
+    
+    - name: Set release tag env
+      run: |
+        echo "TAG_NAME=$(date +'%Y-%m-%d')_$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
+        
+    - uses: ncipollo/release-action@v1
+      with:
+        artifacts: "/srv/dest/lims$(date+'%y%m%d')/vmlinuz,/srv/dest/lims$(date+'%y%m%d')/initrd.img,/srv/dest/lims$(date+'%y%m%d')/root.sfs"
```
Comment (iBug) [inline .github/workflows/build-liimstrap.yml]: Does shell command expansion really work here? I'd rather edit `deploy` script and remove the extra directory altogether.

**Review by iBug — CHANGES_REQUESTED** [review]

File: .github/workflows/build-liimstrap.yml (hunk 1 above)
Comment (RetiredWorld) [reply .github/workflows/build-liimstrap.yml]: Not working. Will changing dest directory break your things? If not, I will try to edit the script.

File: deploy [hunk 2]
```diff
@@ -4,7 +4,7 @@ shopt -s extglob
 
 ROOT="$1"
 DATE="$(date +%y%m%d)"
-DEST="$2/liims$DATE"
+DEST="$2/liimsS"
```
Comment (iBug) [inline deploy]: `liimsS` typo?

Comment (RetiredWorld) [reply deploy]: Fixed, never mind :?

**Comment by iBug** [comment]
The latest commit still doesn't upload build results (see [release](https://github.com/RetiredWorld/liimstrap/releases/tag/2023-02-04_bd6b75)).

**Review by taoky — APPROVED** [review]

**Pull request merged**